packages/platform/types/permissions.ts

```diff
@@ -0,0 +1,3 @@
+import type { PERMISSIONS } from "@calcom/platform-const";
+
+export type PLATFORM_PERMISSIONS = (typeof PERMISSIONS)[number];
```

Comment:
Shall we call it singular `type PLATFORM_PERMISSION` because we select specific one using [number], so it can be only 1 specific permission?

<img width="699" alt="Screenshot 2023-11-30 at 10 11 25" src="https://github.com/calcom/cal.com/assets/42170848/9013f4a8-d7d6-4cac-bfe8-c67cd3b8ecb9">

Reply:
Below we use `PLATFORM_PERMISSIONS[]` so it would read nicer `PLATFORM_PERMISSION[]`

Reply:
indeed good catch